File: static/api3.py
import json, requests
from pprint import pprint

CREEDS_URL = 'http://amp.pharm.mssm.edu/CREEDS/'
response = requests.get(CREEDS_URL + 'api', params={'id':'gene:84'})
if response.status_code == 200:
	pprint(response.json())
	json.dump(response.json(), open('api3_result.json', 'wb'), indent=4)

File: creeds/static/api3.R
library(httr)
library(jsonlite)

CREEDS_URL <- 'http://amp.pharm.mssm.edu/CREEDS/'
response <- GET(paste0(CREEDS_URL, 'api'), query=list(id ='gene:84'))
if (response$status_code == 200){
	response <- fromJSON(httr::content(response, 'text'))
	print(response)
}
